### ISSUE TITLE
Carthage Support. Skipped building zlib-spm due to the error

### ISSUE THREAD

**Issue by pnghitink33b** [watson-developer-cloud/swift-sdk#779]
> carthage update --platform iOS
*** Fetching Starscream
*** Fetching common-crypto-spm
*** Fetching zlib-spm
*** Checking out zlib-spm at "1.1.0"
*** Checking out Starscream at "3.0.4"
*** Checking out common-crypto-spm at "1.1.0"
*** xcodebuild output can be found in /var/folders/67/my163pm1655bw7n35s5f28jr0000gn/T/carthage-xcodebuild.OAxiNt.log
*** Skipped building common-crypto-spm due to the error:
Dependency "common-crypto-spm" has no shared framework schemes for any of the platforms: iOS

>If you believe this to be an error, please file an issue with the maintainers at https://github.com/daltoniam/common-crypto-spm/issues/new
*** Skipped building zlib-spm due to the error:
Dependency "zlib-spm" has no shared framework schemes for any of the platforms: iOS

**Comment by yuandonghua** [comment]
open xcode->preferences->locations->command line tools->Setting up command line tools

**Comment by glennrfisher** [comment]
Hi @pnghitink33b. I started seeing that same error message late last week. I'm not sure what changed to prompt it--perhaps because I updated Carthage on my machine from v0.28.0 to v0.29.0.

There hasn't been a new release of Starscream, but there have been commits to the `master` branch that removed `common-crypto-spm` and `zlib-spm` from their Cartfile. I suspect those changes may fix this warning.

Regardless, this warning can be safely ignored. We don't depend upon any shared framework schemes in the `common-crypto-spm` or `zlib-spm` dependencies. So the SDK should continue working without issue.
